src/lib/env.ts

```diff
@@ -5,8 +5,8 @@ export const isProd = process.env.NODE_ENV === 'production';
 export const isDev = process.env.NODE_ENV === 'development';
 
 export const baseUrl = (req: express.Request): string => {
-    if (process.env.BASE_URL) {
-        return process.env.BASE_URL;
+    if (process.env.base_url) {
```

Comment:
Any reason to downcase these environment variables? I feel like upper case is more conventional/expected.

Reply:
Yeah not sure. So I'm using: https://github.com/guardian/nest-secrets for grabbing them. The convention seems to be lower-case in parameter store but we could change that. I'm not sure I want nest-secrets to uppercase as it's already doing some transformation but I want to avoid it as far as possible (for least surprise).

Reply:
Ah interesting, yeah I agree not doing any transformation is the least surprising approach. This makes sense, knowing that the values are coming directly from parameter store.